### PR TITLE
Internal docs for archive db apps

### PR DESCRIPTION
**Pull request by psteckler**
Add internal documentation for `archive_blocks`, `extract_blocks`, and `missing_blocks_auditor`.

### CODE REVIEW

**Comment by psteckler** [comment]
!ci-build-me

**Comment by psteckler** [comment]
!ci-build-me

File: helm/cron_jobs/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Replayer cron jobs
+==================
+
+There are replayer cron jobs for mainnet, devnet, and berkeley. These
+jobs are run daily, to replayer a day's worth of transactions.
```
Comment (barriebyron) [inline helm/cron_jobs/README.md:5]:
```suggestion
There are replayer cron jobs for Mainnet, Devnet, and Berkeley. These
jobs are run daily to replay a day's worth of transactions.
```

File: helm/cron_jobs/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Each cron job downloads the most recent archive dump corresponding to
+a network, and loads the data into Postgresql. That results in an
+archive database. The most recent replayer checkpoint file is
+downloaded, which provides the starting point for the replayer. When
```
Comment (barriebyron) [inline helm/cron_jobs/README.md:10]:
```suggestion
a network, and loads the data into PostgreSQL. That results in an
archive database. The most recent replayer checkpoint file is
downloaded, which provides the starting point for the replayer. When
```

File: helm/cron_jobs/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+the replayer runs, it creates new checkpoint files every 50
+blocks. When the replayer finishes, it uploads the most recent
+checkpoint file, so it can be used in the following day's run. If
+there are any errors, the replayer logs are also uploaded.
+
+There is a separate checkpoint file bucket for each network.  Bot the
+checkpoint files and error files for a given network are uploaded to
+the same bucket.
```
Comment (barriebyron) [inline helm/cron_jobs/README.md:18]:
```suggestion
There is a separate checkpoint file bucket for each network.  Both the
checkpoint files and error files for a given network are uploaded to
the same bucket.
```

File: src/app/missing_blocks_auditor/missing_blocks_auditor.ml [hunk 4]
```diff
@@ -1,4 +1,13 @@
-(* missing_blocks_auditor.ml -- report missing blocks from an archive db *)
+(* missing_blocks_auditor.ml *)
+
+(* missing_blocks_auditor looks for blocks without parent blocks in an
+   archive database.
+
+   The app also looks for blocks marked as pending that are lower
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply src/app/missing_blocks_auditor/missing_blocks_auditor.ml:6]: "The app" is the ML file that is the missing blocks auditor app?

File: helm/cron_jobs/README.md (hunk 3 above)
Comment (psteckler) [reply helm/cron_jobs/README.md]: Fixed in my own commit.

File: helm/cron_jobs/README.md (hunk 1 above)
Comment (psteckler) [reply helm/cron_jobs/README.md]: Fixed in my own commit. I'm keeping the comma, though :-)

File: src/app/missing_blocks_auditor/missing_blocks_auditor.ml (hunk 4 above)
[the remark this answers is not in the document]
Comment (psteckler) [reply src/app/missing_blocks_auditor/missing_blocks_auditor.ml]: The .ml file gets compiled to an executable program. "The app" is the executable.

**Comment by psteckler** [comment]
!ci-build-me

File: helm/cron_jobs/README.md (hunk 2 above)
Comment (psteckler) [reply helm/cron_jobs/README.md]: Fixed in my commit.

**Review by ghost-not-in-the-shell — APPROVED** [review]

**Comment by psteckler** [comment]
!ci-build-me

**Pull request merged**